handlers/bootstrap.go

```diff
@@ -130,6 +130,17 @@ func buildContainerConfig(containerCmd []string, machine *client.Machine, imgRep
 
 	volConfig := map[string]struct{}{"/var/run/docker.sock": {}}
 	envVars := []string{"CATTLE_PHYSICAL_HOST_UUID=" + machine.ExternalId}
+	for key, value := range machine.Labels {
+		hostLabel := ""
+		switch value.(type) {
+		case string:
+			hostLabel = value.(string)
+		default:
+			continue
+		}
+		hostLabelPair := key + "=" + hostLabel
```

Comment:
I would assume this is just a general env labeling system we will support in the future that may not actually be a hostLabel.  I don't think it matters much here but the variable probably should be just a label and not actually tied to a host.